python/cryptography/security/insufficient-dsa-key-size.yaml

```diff
@@ -8,6 +8,9 @@ rules:
   - metavariable-comparison:
       metavariable: $SIZE
       comparison: $SIZE < 2048
+  - focus-metavariable: $SIZE
+  fix: |
```

Comment:
I  understand we want a min key size, but I would think we should also get the folks over to ECDSA. Why not have the fix actually move them to it?